.codeclimate.yml

```diff
@@ -32,6 +32,6 @@ exclude_paths:
   - assets/js/jquery-*
   - assets/js/prettyPhoto/*
   - assets/js/round/*
-  - assets/js/select2/*
```

Comment:
The select2 folder doesn't exist any more, though in this PR. It's been replaced with a selectWoo folder.